src/commands/periscope/models/RetinaDownloadConfig.ts

```diff
@@ -0,0 +1,3 @@
+export interface RetinaDownloadConfig {
```

Comment:
we can change this to GithubReleaseConfig, I thought it was specific to retina, but it is both for draft and retina. Just a minor name change please.

Reply:
Thanks for this, given its gh retina config we can discuss renaming for latter because draft work is the whole other PR which has full drraft hooked.